README.md

```diff
@@ -1,36 +1,21 @@
 ![IBM Tutorials Logo](./images/tutorialslogo.png)
+<div align = 'center'>
 <hr noshade>
-<h1>IBMdotcom Tutorials</h1>
-<p><i>Cutting-edge AI, ML, NLP, and Data Governance Tutorials and Explainers</i></p>
-
 <p>Welcome to the centralized repository for all technical explainers, tutorials, and demos developed by inbound ibm.com technical writing team. This repository focuses on cutting-edge technologies in the fields of:</p>
```

Comment:
I would remove the oxford comma to align with the rest of the readme. Adding "by THE inbound ibm.com technical writing team" also sounds cleaner.